snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/compose/SnapshotVariantProvider.kt

```diff
@@ -52,11 +61,15 @@ fun SnapshotVariantProvider(
     }
   }
 
+  val layoutDirection =
```

Comment:
```suggestion
  val layoutDirection = TextUtilsCompat.getLayoutDirectionFromLocale(locale)
```

Reply:
This gives an int which is not compatible with the `LayoutDirection` the `LocalLayoutDirection` compositionLocal is expecting. Will see if I can work around it as this is a useful helper.